src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -496,33 +504,6 @@ public void canCheckOutViaProxy() {
       loan.getString("proxyUserId"), is(james.getId()));
   }
 
-  @Test
-  public void cannotCheckOutWhenLoanPolicyDoesNotExist() {
```

Comment:
I think this situation is still possible, if the policy is deleted after the rules are updated. Do you agree?

Reply:
sure, returned it back